README.md

```diff
@@ -1,8 +1,304 @@
-# RFC
+# Amethyst RFCs
+[Amethyst RFCs]: #amethyst-rfcs
 
-This is the repository where we work and keep track of the all RFC documents related to Amethyst.
-Having this repository allows people to participate in decision making by opening a pull request with a new document.
+- [Discussing RFCs](https://github.com/amethyst/rfcs/issues)
+- [Accepted RFCs](https://github.com/amethyst/rfcs/tree/master/accepted)
+- [RFCs Awaiting Implementation](https://github.com/amethyst/rfcs/issues?utf8=✓&q=label%3A%22RFC%3A+Accepted%22+-label%3A%22Status%3A+Working%22+)
```

Comment:
I'm a bit unsure here. I think it makes sense to have the issues in the repositories they need to be implemented in, but at the same time that makes it hard to link to them (unless we make a sub-list with the repos).

Reply:
I would link each sub-issue in the tracking issue, especially since they may across repo boundaries.

Reply:
Okay, I like that.